server/homebrew.api.js

```diff
@@ -91,7 +98,7 @@ const updateBrew = (req, res)=>{
 			brew.updatedAt = new Date();
 
 			if(req.account) {
-				brew.authors = _.uniq(_.concat(brew.authors, req.account.username));
+				brew.authors = _.uniq(_.concat(updateBrew.authors.filter((a)=>a.length > 0), req.account.username));
```

Comment:
What is this doing? Just making sure you can't add authors with an empty string for the name?

Could this be handled in the UI so you can't add `''` as a username anyway?

Reply:
Yeah we can handle it in the UI as well, but I don't want it to be possible on the API either.

Reply:
That's fair. Just so the UI doesn't inply you can do something but the API does something else.